x11-misc/idesk/idesk-0.7.5-r3.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Utility to place icons on the root window"
+HOMEPAGE="http://idesk.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.bz2"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ppc ~ppc64 ~sparc ~x86"
+IUSE=""
+
+RDEPEND=">=media-libs/imlib2-1.4[X]
+	media-libs/freetype
+	dev-libs/libxml2
+	dev-libs/glib:2
+	x11-libs/pango
+	x11-libs/gtk+:2
+	media-libs/libart_lgpl
+	x11-libs/startup-notification"
+DEPEND="${RDEPEND}
```

Comment:
yes please, I forgot to check for this - this is part of the "unofficial" Gentoo style (first unconditional, then conditional deps, always sorted within blocks)